.github/workflows/check-projects.yml

```diff
@@ -0,0 +1,99 @@
+name: check-projects
+on:
+  pull_request:
+    branches:
+      - master
+
```

Comment:
Add `permissions` to reduce the privilege of GITHUB_TOKEN.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication